MAUI/ImageEditor/customview.md

```diff
@@ -132,7 +132,7 @@ N> This is common for Shape, Text and CustomView annotations.
 
 ## Annotation selected event
```

Comment:
Change file name, custom-view update in toc file also

Reply:
Changed file name and updated in toc file